src/runtime_src/core/edge/skd/sk_daemon.cpp

```diff
@@ -57,13 +57,10 @@ xrt::skd* skd_inst = nullptr;
 /* Define a signal handler for the child to handle signals */
 static void sigLog(const int sig)
```

Comment:
the signal handler now does nothing but just logging? do you need to set some flag here?

Reply:
The SIGTERM signal will cause down_interruptible to return with -EINTR, which will then exit the process.
No flag need to be set.

Reply:
Instead of upping the sem, we send signal? This is a temp hack, right?